certbot/main.py

```diff
@@ -405,6 +406,53 @@ def _init_le_client(config, authenticator, installer):
 
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
+def _cert_path_to_lineage(config):
```

Comment:
Except maybe not since `_search_lineages` will do the extra unnecessary work of loading the lineage. But either way it should probably have those checks and exceptions, and be over there.